examples/getting-started-movielens/03-Training-with-TF.ipynb

```diff
@@ -2,7 +2,7 @@
  "cells": [
```

Comment:
<p>Line #5.        prediction_tasks=mm.BinaryClassificationTask(target_column),</p><p> </p><p>we  no longer use  <code>mm.BinaryClassificationTask</code>  instead we use  <code>prediction_tasks=mm.BinaryOutput(target_column) </code>.</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/Merlin/pull/779/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Reply:
<p>change made</p>

Reply:
<p>unfortunately, the serving operator doesn't work if I use <code>mm.BinaryOutput</code></p><p> </p><pre class="ql-syntax" spellcheck="false"> ---------------------------------------------------------------------------
InferenceServerException                  Traceback (most recent call last)
Cell In [11], line 9
      3 outputs = [
      4     grpcclient.InferRequestedOutput(col)
      5     for col in ["rating/binary_classification_task"]
      6 ]
      8 with grpcclient.InferenceServerClient("localhost:8001") as client:
----&gt; 9     response = client.infer("predicttensorflow", inputs, request_id="1", outputs=outputs)

File /usr/local/lib/python3.8/dist-packages/tritonclient/grpc/__init__.py:1431, in InferenceServerClient.infer(self, model_name, inputs, model_version, outputs, request_id, sequence_id, sequence_start, sequence_end, priority, timeout, client_timeout, headers, compression_algorithm)
   1429     return result
   1430 except grpc.RpcError as rpc_error:
-&gt; 1431     raise_error_grpc(rpc_error)

File /usr/local/lib/python3.8/dist-packages/tritonclient/grpc/__init__.py:62, in raise_error_grpc(rpc_error)
     61 def raise_error_grpc(rpc_error):
---&gt; 62     raise get_error_grpc(rpc_error) from None

InferenceServerException: [StatusCode.INVALID_ARGUMENT] unexpected inference output 'rating/binary_classification_task' for model 'predicttensorflow'
</pre>

Reply:
@radekosmulski if we can figure out what's the correct output name in your config file (please check it out) you can replace it in the `for col in ["rating/binary_classification_task"]` line here. if config files do not show the output name, we can checkout from model.